README.md

```diff
@@ -1,38 +1,169 @@
-## PROJECT NAME
+# dotnet-mathematical-program
+==========
 
-A short description of what this project does.
+[![](https://img.shields.io/nuget/v/Anexia.MathematicalProgram "NuGet version badge")](https://www.nuget.org/packages/Anexia.MathematicalProgram)
+[![](https://github.com/anexia/dotnetcore-mathematical-program/actions/workflows/test.yml/badge.svg?branch=main "Test status")](https://github.com/anexia/dotnetcore-mathematical-program/actions/workflows/test.yml)
 
-## Goals
+This library allows you to build and solve linear programs and integer linear programs in a very handy way.
+The implementation uses Google´s GLOP linear solver for linear programs and optionally, the Coin-OR CBC branch and cut
+solver
+or the Gurobi solver for integer linear programs via the [Google OR-Tools](https://developers.google.com/optimization)
+API.
 
-It is a good idea to provide a mission statement for your project, enshrining
-what the project wants to accomplish so that as more people join your project
-everyone can work in alignment.
+## Installation
 
-### Installation
+- Install latest package of `Anexia.MathematicalProgram` via nuget
```

Comment:
Install the latest version of the `Anexia.MathematicalProgram` package via nuget